package.json

```diff
@@ -8,20 +8,30 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@tailwindcss/forms": "^0.3.3",
+    "@vue-hero-icons/solid": "^1.7.2",
```

Comment:
We don't need to install a packge just for an icon, let's uninstall this and maybe find an svg or don't put anything at all